docs/topics/plugin-intro.adoc

```diff
@@ -1,11 +1,22 @@
 // Module included in the following assemblies:
 //
 // * docs/plugin-guide/master.adoc
+// * docs/vscode-plugin-guide/master.adoc
 
+
+ifdef::plugin-guide[]
 [id='plugin-intro_{context}']
 = About the {PluginBookName}
 
 This guide is for engineers, consultants, and others who want to use the {PluginName} for the {ProductName} ({ProductShortName}) to assist with migrating applications.
+endif::[]
+
+ifdef::vscode-plugin-guide[]
+[id='vscode-plugin-intro_{context}']
+= About the {VscodePluginBookName}
+
+This guide is for engineers, consultants, and others who want to use the {VscodePluginName} ({VscodePluginShortName}) extension with the {ProductName} ({ProductShortName}) to assist with migrating applications.
```

Comment:
Honestly, I think you should replace all these VS Code attributes with just the name. You are never ever going to have to update the attributes themselves, so nothing is gained. It just makes it harder to type the name of the plugin. In fact, the attributes are more likely to have typos than the actual names themselves.